autoload/codefmt/formatterhelpers.vim

```diff
@@ -87,21 +88,35 @@ endfunction
 " the tool for range formatting and post a URL for that feature request above
 " code that calls it.
 "
-" @throws ShellError if the {cmd} system call fails
+" If [ignoreerrors] is nonzero, the syscall ignores errors. This can be helpful
+" for formatters that return nonzero results for reasons unrelated to
+" formatting. If [skipfirstnlines] is set to a nonzero number N, the first
+" N lines of the formatter output are trimmed. This can be used to trim
+" always-present headers.
+"
+" @throws ShellError if the {cmd} system call fails (and [ignoreerrors] is 0)
 " @throws WrongType
 function! codefmt#formatterhelpers#AttemptFakeRangeFormatting(
-    \ startline, endline, cmd) abort
+    \ startline, endline, cmd, ...) abort
   call maktaba#ensure#IsNumber(a:startline)
   call maktaba#ensure#IsNumber(a:endline)
 
+  let l:ignoreerrors = a:0 >= 1 ? a:1 : 0
```

Comment:
Fun fact: You can also do `let l:ignoreerrors = get(a:, 1, 0)` for the same effect, slightly more cryptic but a bit more literal once you're used to the syntax.

Either way works fine here though, and I'm focused on getting this PR merged quickly after all the delays. =)